base/Dockerfile

```diff
@@ -35,7 +35,6 @@ RUN apt update -y && \
         ca-certificates
 
 COPY lnls-get-n-unpack.sh /usr/local/bin/lnls-get-n-unpack
-COPY lnls-run.sh /usr/local/bin/lnls-run
```

Comment:
This commit message is wrong. This script isn't executed during any build step.

Reply:
Indeed. I was willing to write in the affirmative tense, but made the complement subset smaller than it actually is. Thanks.